source/extensions/filters/http/cache/cache_filter.h

```diff
@@ -86,8 +88,9 @@ class CacheFilter : public Http::PassThroughFilter,
   // Updates filter_state_ and continues the encoding stream if necessary.
   void finalizeEncodingCachedResponse();
 
+  CacheFilterConfig config_;
+  HttpCachePtr cache_ = nullptr;
```

Comment:
Sorry, this is my fault - I said rather than explicit initialization in the constructor it could be done here, but now that it's here I realize HttpCachePtr is a shared_ptr so it already default-initializes to nullptr. (If anything =nullptr here harms readability because it makes it look like it's a regular pointer rather than the shared pointer that it is.)